modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -71,7 +74,7 @@ public static Map<String, Object> getActionVariables(
 			).put(
 				"originalObjectEntry",
 				() -> {
-					if (payloadJSONObject.has("originalObjectEntry")) {
+					if (payloadJSONObject.has("original" + suffix)) {
```

Comment:
@p-albuquerque can we inline here? 
```
if (payloadJSONObject.has("original" + _getSuffix(objectDefinition, systemObjectDefinitionMetadataRegistry))) {
```

Or put the `_getSuffix` right before the `if`:

```
"originalObjectEntry",
() -> {
       String suffix = _getSuffix(objectDefinition, systemObjectDefinitionMetadataRegistry);

       if (payloadJSONObject.has("original" + suffix)) {
```

Reply:
I will try the second way. I believe the CI was denied the first suggestion, but I will check again